pages/index.tsx

```diff
@@ -58,6 +60,12 @@ const Home: NextPageWithLayout = () => {
     router.push(session?.cancel_url || 'https://awell.health')
   }
 
+  const shouldRedirect =
+    (session?.status === HostedSessionStatus.Completed &&
+      !isNil(session?.success_url)) ||
+    (session?.status === HostedSessionStatus.Expired &&
+      !isNil(session?.cancel_url))
+
```

Comment:
[Q] @mohsinht What if I define a success URL but no cancel URL or vice versa (i.e. one of both is set)? Don't we need a  `shouldRedirectIfSuccess` and `shouldRedirectIfCancel` instead? Based on the code alone it is difficult to predict what will happen so maybe good if you test it.

**Expected behavior in case a success URL is set but no cancel URL:**
- Redirect when session is completed
- Show cancel page when session is expired

**Expected behavior in case a cancel URL is set but no success URL:**
- Show success page when session is completed
- Redirect when session is expired

Reply:
I've tested it as well

```ts
const shouldRedirect =
    (session?.status === HostedSessionStatus.Completed &&
      !isNil(session?.success_url)) ||
    (session?.status === HostedSessionStatus.Expired &&
      !isNil(session?.cancel_url))
```

**Case 1: When `session = { status: 'completed', cancel_url: 'abc' }`** 

```ts
const shouldRedirect =
    (true &&
      false) ||
    (false &&
      true)

// result FALSE
```

**Case 2: When `session = { status: 'completed', success_url: 'abc' }`** 

```ts
const shouldRedirect =
    (true &&
      true) ||
    (false &&
      false)

// result TRUE
```

**Case 3: When `session = { status: 'expired', success_url: 'abc' }`** 

```ts
const shouldRedirect =
    (false &&
      true) ||
    (true &&
      false)

// result FALSE
```

**Case 4: When `session = { status: 'expired', cancel_url: 'abc' }`** 

```ts
const shouldRedirect =
    (false &&
      false) ||
    (true &&
      true)

// result TRUE
```